src/extension.ts

```diff
@@ -1876,7 +1876,13 @@ async function setup(context: vscode.ExtensionContext, progress?: ProgressHandle
         vscode.commands.registerCommand('cmake.outline.editCacheUI', () => runCommand('editCacheUI')),
         vscode.commands.registerCommand('cmake.outline.cleanRebuildAll', () => runCommand('cleanRebuildAll')),
         // Commands for outline items
-        vscode.commands.registerCommand('cmake.outline.configure', (what: ProjectNode) => runCommand('configure', what.folder, false, what.sourceDirectory)),
+        vscode.commands.registerCommand('cmake.outline.configure', async (what: object) => {
```

Comment:
It'd be good if we could restrict this type more than just `object`, something that only allows for `ProjectNode` or `SourceFileNode`.

Reply:
Thanks for review.
I've changed type annotation. Is it better?

Reply:
Yes! Everything looks good. Please add a Changelog entry and then I will approve and merge!

Reply:
Done